servconf.c

```diff
@@ -2084,6 +2084,31 @@ parse_server_config(ServerOptions *options, const char *filename, Buffer *conf,
 		fatal("%s: terminating, %d bad configuration options",
 		    filename, bad_options);
 	process_queued_listen_addrs(options);
+
+#ifdef WINDOWS
+	/* convert the users, user groups to lower case */
```

Comment:
Add a TODO comment like the following:
TODO - Refactor this into a platform specific post-read config processing routine.